Makefile

```diff
@@ -2,24 +2,17 @@
 
 all:
 	@echo ""
-	@echo "  install: install dep (assumes go is installed)"
-	@echo "  deps: grab dependencies using dep"
+	@echo "  deps: grab dependencies using go modules"
 	@echo "  test: test prebid-server (via validate.sh)"
 	@echo "  build: build prebid-server"
 	@echo "  image: build docker image"
 	@echo ""
 
-.PHONY: install deps test build image
```

Comment:
Any reason for removing the `PHONY` targets? Generally, it is recommended that any target in the `makefile` that doesn't result into an actual file with the same name should be listed as a `PHONY` target.

More info about `PHONY` targets: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

Reply:
Oh, you are right, I just wanted to remove the deps part :)